.editorconfig

```diff
@@ -14,3 +14,6 @@ indent_size = 2
 
 [*.yml]
 indent_size = 2
+
+[*.yaml]
```

Comment:
No biggie, for future reference this could be `*.{yml,yaml}`  above

Reply:
https://github.com/guardian/grid/pull/2094/files 😄

Reply:
💯